nl_NL/LC_MESSAGES/tickets.po

```diff
@@ -197,7 +203,7 @@ msgid "TICKET_NAME_colors_promo_ticket"
 msgstr "Colors! Promoticket"
 
 msgid "TICKET_DESC_colors_promo_ticket"
-msgstr "You won the Sudomemo x Colors! June 2020 Weekly Topic!"
+msgstr "Jij hebt de Sudomemo x Colors gewonnen! Weekly topic juni 2020!"
```

Comment:
Don't move the !, `Colors! Live` is the name of the game, in this case shortened to just `Colors!`. The first letter of "Juni" should be capitalized.